project/profiles/capacity4more/modules/c4m/features/c4m_features_footer/theme/c4m-features-footer-bar-project.tpl.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * Template for presenting OG project pages footer.
+ */
+?>
+
+<div class="row">
+  <ul class="c4m-footer-bar clearfix">
```

Comment:
when/if removing the `row` class from the parent element, check if we still need the `clearfix` in here